lib/app/features/user/providers/user_metadata_notifier.dart

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: ice License 1.0
+
+import 'package:ion/app/features/nostr/model/file_metadata.dart';
+import 'package:ion/app/features/nostr/model/media_attachment.dart';
+import 'package:ion/app/features/nostr/providers/nostr_notifier.dart';
+import 'package:ion/app/features/nostr/providers/nostr_upload_notifier.dart';
+import 'package:ion/app/features/user/model/user_metadata.dart';
+import 'package:ion/app/services/media_service/media_service.dart';
+import 'package:riverpod_annotation/riverpod_annotation.dart';
+
+part 'user_metadata_notifier.g.dart';
+
+@riverpod
+class UserMetadataNotifier extends _$UserMetadataNotifier {
```

Comment:
Imho, the naming is poor in this file. `UserMetadataNotifier` and it's method `send` doesn't say what exactly they do. I'd name it something like `EditProfileNotifier`.